packages/n4s/src/rules/isNegative/index.js

```diff
@@ -0,0 +1,8 @@
+function isNegative(value, arg1) {
+  if (!isNaN(Number(value))) {
+    return Number(value) < 0;
+  }
+  return false;
+}
+
+export default isNegative;
```

Comment:
Adding this will allow you to completely remove the isPositive file and its tests.

```suggestion
}

isNegative.negativeForm = 'isPositive';

export default isNegative;
```